### ISSUE TITLE
Do: VersionChecker

### ISSUE THREAD

**Issue by Adaptivity** [F32Organization/Faithful32-1.7.10#109]
https://github.com/Dynious/VersionChecker/tree/master/src/main/resources/assets/versionchecker/textures/gui
Should be easy.

**Comment by DZCreeper** [comment]
I will do that soon, maybe even today. Speaking of version checker, that's something I really want to get working for Faithful 32. If anyone knows a way to accomplish this short of writing a custom mod, let me know.

**Comment by Evolution0** [comment]
A bit of an old issue but.. have you considered altering the build script to superimpose a version and build date onto the mojang logo? Should just take a bit of string manipulation in bash and a bit of imagemagick.

I've worked with imagemagick in scripts before and its not too hard (though I made a last.fm "now listening" forum signature generator) the idea of superimposing text onto an image was the same.

If you want, send me the build script you use on the server and I'll fiddle with it locally to do this.
